litex/SoC-CircuitPython.py

```diff
@@ -253,18 +251,15 @@ def __init__(self, sys_clk_freq=int(48e6), toolchain="trellis", **kwargs):
 
         # The litex SPI module supports memory-mapped reads, as well as a bit-banged mode
         # for doing writes.
-        spi_pads = platform.request("spiflash4x")
-        self.submodules.lxspi = spi_flash.SpiFlashDualQuad(spi_pads, dummy=6, endianness="little")
```

Comment:
I have not mapped endianness or dummy, @gregdavill do you remember what was the purpose of dummy?